sim_pipeline/image_simulation.py

```diff
@@ -2,10 +2,13 @@
 import galsim
 from astropy.visualization import make_lupton_rgb
 
+
 def simulate_image(lens_class, band, num_pix, add_noise=True, observatory='LSST', **kwargs):
     """
+    Creates an image of a selected lens with noise.
 
-    :param lens_class: class object containing all information of the lensing system (e.g., GGLens())
+    :param lens_class: class object containing all information of the lensing system (e.g., 
+    GGLens())
```

Comment:
there should be one white space inline, otherwise the documentation is not formatted well.